vumi/persist/riak_base.py

```diff
@@ -0,0 +1,208 @@
+"""Basic tools for building a Riak manager."""
+
+import json
+
+from riak import RiakClient
+
+
+def to_unicode(text, encoding='utf-8'):
+    if text is None:
+        return text
+    if isinstance(text, tuple):
+        return tuple(to_unicode(item, encoding) for item in text)
+    if not isinstance(text, unicode):
```

Comment:
Super minor nitpick... but I find this logic a bit confusing. Why not...
```python
if text is None or isinstance(text, unicode):
    return text
...
return text.decode(encoding)
```
... just the `is not` and then the sort of "fall-through" of the unicode case I found unclear.